utils/utils.go

```diff
@@ -27,6 +27,9 @@ const (
 	MainnetChainID = "evmos_9001"
 	// TestnetChainID defines the Evmos EIP155 chain ID for testnet
 	TestnetChainID = "evmos_9000"
+	// TestingChainID defines the Evmos EIP155 chain ID for testing purposes
+	// like the local node.
+	TestingChainID = "evmos_9002"
```

Comment:
also seems unrelated to this PR?

Reply:
It is! Before it was falling back to the default mainnet chain ID for the configuration because it was not found in the `InitializeAppConfiguration`. Now I enforced every chain id to be registered to avoid silent fallback of configuration